index.html

```diff
@@ -32,13 +34,124 @@ <h5>
       LET’S CONNECT
     </h5>
     <ul>
-      <li><i class="fa-brands fa-twitter socialMedia"></i></li>
-      <li><i class="fa-brands fa-linkedin-in socialMedia"></i></li>
-      <li><i class="fa-solid fa-m socialMedia"></i></li>
-      <li><i class="fa-brands fa-github socialMedia"></i></li>
-      <li><i class="fa-regular fa-hand-peace socialMedia"></i></li>
+      <li><a href="#"><i class="fa-brands fa-twitter socialMedia"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-linkedin-in socialMedia"></i></a> </li>
+      <li><a href="#"><i class="fa-solid fa-m socialMedia"></i></a></li>
+      <li><a href="#"><i class="fa-brands fa-github socialMedia"></i></a></li>
+      <li><a href="#"><i class="fa-regular fa-hand-peace socialMedia"></i></a></li>
     </ul>
   </section>
+  <section class="work">
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio0.png" alt="" srcset="">
+      <h2>Tonic</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
```

Comment:
- [ ] The first reviewer asked that you should work on the Typography of the elements in this section. **_Type color_** is an element of typography that describes how dense or heavy the text appears on the page. The description text for each card has a color of `#344563`. It would be nice to implement this so that your design is identical to that of the Figma template that you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/219474064-aa0ee76c-edc6-436a-b0ce-137604f392e0.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/219474162-86d7ce60-525d-4ac1-9b78-b0142c7cf673.png)